### ISSUE TITLE
DXF export spline location

### ISSUE THREAD

**Issue by sethfischer** [CadQuery/cadquery#1263]
CadQuery: 2.2.0
Python: 3.9.5
ezdxf: 1.0.1
OS: Ubuntu 20.04.5

---

The DXF generated by the following has its splines offset. A bug appears to be in `cadquery.occ_impl.exporters.dxf._dxf_spline`.

```python
"""Export text as DXF."""

import cadquery as cq
from cadquery import exporters

text = cq.Workplane().text("cQ", 5, 1)

exporters.dxf.exportDXF(text, "output.dxf")
```
`output.dxf` viewed in LibreCad 2.1.3:

![cq-spline-offset](https://user-images.githubusercontent.com/2492233/216876888-565da039-cd8c-47b5-aef9-b971ca3bd50f.png)

**Comment by lorenzncode** [comment]
I experimented with this and a transformation may be required perhaps something like the following in `_dxf_spline`:
```spline.Transform(plane.rG.wrapped.Trsf()*adaptor.Trsf())```

and a similar transformation in the other dxf methods.

Testing with:
```
text = cq.Workplane("XZ").text("CQ", 5, 1)
cq.exporters.dxf.exportDXF(text.section(), "output.dxf")
```
![image](https://user-images.githubusercontent.com/16394272/219895941-dbc7a2b7-dc01-446c-a8db-ecadcce74ea6.png)

**Comment by adam-urbanczyk** [comment]
Solved in #1267
